### PR TITLE
add day npow mint limit

### PR DESCRIPTION
**Pull request by xcaptain**

### CODE REVIEW

File: pallets/operation/src/lib.rs [hunk 1]
```diff
@@ -72,6 +72,7 @@ pub mod pallet {
         type MinimumBurnedDPR: Get<BalanceOf<Self>>;
         type CreditInterface: CreditInterface<Self::AccountId, BalanceOf<Self>>;
         type UserPrivilegeInterface: UserPrivilegeInterface<Self::AccountId>;
+        type NpowMintDayLimit: Get<BalanceOf<Self>>;
```
Comment (jerry-yu) [inline pallets/operation/src/lib.rs:75]: use storage_value ,and make it value can only be modify by root

File: pallets/operation/src/lib.rs [hunk 2]
```diff
@@ -218,6 +220,11 @@ pub mod pallet {
     #[pallet::storage]
     pub(super) type StorageVersion<T> = StorageValue<_, Releases>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn npow_day_minted_dpr)]
+    pub type NpowDayMintedDPR<T: Config> =
+        StorageMap<_, Blake2_128Concat, EraIndex, BalanceOf<T>, OptionQuery>;
```
Comment (jerry-yu) [inline pallets/operation/src/lib.rs:226]: use StorageValue<(EraIndex,Balanceof<T>)> is enough for calculating current era limit.
if using StorageMap , need clear old era's record

File: pallets/operation/src/lib.rs (hunk 1 above)
Comment (xcaptain) [reply pallets/operation/src/lib.rs]: fixed, please take a look again(for simplicity I reused the weights).

File: pallets/operation/src/lib.rs (hunk 2 above)
Comment (xcaptain) [reply pallets/operation/src/lib.rs]: Good point, changed to storagevalue now.

**Review by jerry-yu — APPROVED** [review]

**Pull request merged**